src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
```

Comment:
there should be no inheritance in this HW
if you need to call a method of a class, you should create an object of this class